static/app/utils/fields/index.ts

```diff
@@ -0,0 +1,574 @@
+  DEVICE_BRAND = 'device.brand',
+  DEVICE_CHARGING = 'device.charging',
+  DEVICE_FAMILY = 'device.family',
+  DEVICE_LOCALE = 'device.locale',
+  DEVICE_MODEL_ID = 'device.model_id',
+  DEVICE_NAME = 'device.name',
+  DEVICE_ONLINE = 'device.online',
+  DEVICE_ORIENTATION = 'device.orientation',
+  DEVICE_SIMULATOR = 'device.simulator',
+  DEVICE_UUID = 'device.uuid',
+  DIST = 'dist',
+  ENVIRONMENT = 'environment',
+  ERROR_HANDLED = 'error.handled',
+  ERROR_MECHANISM = 'error.mechanism',
+  ERROR_TYPE = 'error.type',
+  ERROR_UNHANDLED = 'error.unhandled',
+  ERROR_VALUE = 'error.value',
+  EVENT_TIMESTAMP = 'event.timestamp',
+  EVENT_TYPE = 'event.type',
+  FIRST_RELEASE = 'firstRelease',
+  FIRST_SEEN = 'firstSeen',
+  GEO_CITY = 'geo.city',
+  GEO_COUNTRY_CODE = 'geo.country_code',
+  GEO_REGION = 'geo.region',
+  HAS = 'has',
+  HTTP_METHOD = 'http.method',
+  HTTP_REFERER = 'http.referer',
+  HTTP_STATUS_CODE = 'http.status_code',
+  HTTP_URL = 'http.url',
+  ID = 'id',
+  IS = 'is',
+  ISSUE = 'issue',
+  LAST_SEEN = 'lastSeen',
+  LEVEL = 'level',
+  LOCATION = 'location',
+  MESSAGE = 'message',
+  OS = 'os',
+  OS_BUILD = 'os.build',
+  OS_KERNEL_VERSION = 'os.kernel_version',
+  PLATFORM = 'platform',
+  PLATFORM_NAME = 'platform.name',
+  PROJECT = 'project',
+  RELEASE = 'release',
+  RELEASE_BUILD = 'release.build',
+  RELEASE_PACKAGE = 'release.package',
+  RELEASE_STAGE = 'release.stage',
+  RELEASE_VERSION = 'release.version',
+  SDK_NAME = 'sdk.name',
+  SDK_VERSION = 'sdk.version',
+  STACK_ABS_PATH = 'stack.abs_path',
+  STACK_COLNO = 'stack.colno',
+  STACK_FILENAME = 'stack.filename',
+  STACK_FUNCTION = 'stack.function',
+  STACK_IN_APP = 'stack.in_app',
+  STACK_LINENO = 'stack.lineno',
+  STACK_MODULE = 'stack.module',
+  STACK_PACKAGE = 'stack.package',
+  STACK_RESOURCE = 'stack.resource',
+  STACK_STACK_LEVEL = 'stack.stack_level',
+  TIMESTAMP = 'timestamp',
+  TIMESTAMP_TO_DAY = 'timestamp.to_day',
+  TIMESTAMP_TO_HOUR = 'timestamp.to_hour',
+  TIMES_SEEN = 'timesSeen',
+  TITLE = 'title',
+  TRACE = 'trace',
+  TRACE_PARENT_SPAN = 'trace.parent_span',
+  TRACE_SPAN = 'trace.span',
+  TRANSACTION = 'transaction',
+  TRANSACTION_DURATION = 'transaction.duration',
+  TRANSACTION_OP = 'transaction.op',
+  TRANSACTION_STATUS = 'transaction.status',
+  USER = 'user',
+  USER_DISPLAY = 'user.display',
+  USER_EMAIL = 'user.email',
+  USER_ID = 'user.id',
+  USER_IP = 'user.ip',
+  USER_USERNAME = 'user.username',
+}
+
+export enum FieldValueType {
+  BOOLEAN = 'boolean',
+  DATE = 'date',
+  DURATION = 'duration',
+  INTEGER = 'integer',
+  NUMBER = 'number',
+  PERCENTAGE = 'percentage',
+  STRING = 'string',
+}
+
+export interface FieldDefinition {
+  kind: FieldKind;
+  valueType: FieldValueType;
+  deprecated?: boolean;
+  desc?: string;
+}
+
+export const FIELDS: Record<string, FieldDefinition> = {
+  [FieldKey.AGE]: {
+    desc: 'The age of the issue in relative time',
```

Comment:
Should all of the `desc`'s be wrapped in `t()`

Reply:
we don't do any translations currently right, just in case for the future?

Reply:
so when you wrap it in `t()` it will get included in strings that can be translated. So next time we do translations this will be in the set of strings to translate